test/acceptance.hs

```diff
@@ -277,8 +278,11 @@ getAuthorizePage
     -> Maybe (UserID, Scope)
     -> (ClientID, Scope)
     -> ExceptT String IO ByteString
-getAuthorizePage base_uri user_m (client, scope) = do
+getAuthorizePage base_uri user_m (client, req_scope) = do
     let opts = defaults & header "Accept" .~ ["text/html"]
+                        & param "response_type" .~ ["code"]
+                        & param "client_id" .~ [T.decodeUtf8 $ review clientID client]
+                        & param "scope" .~ [T.decodeUtf8 . scopeToBs $ req_scope]
```

Comment:
You could maybe compose your prisms with this if that made you happier.

https://hackage.haskell.org/package/lens-4.11/docs/src/Data-Text-Strict-Lens.html#utf8